app/assets/stylesheets/cards.css.scss

```diff
@@ -0,0 +1,74 @@
+$flickr-pink: #ff2e92;
+$flickr-blue: #007bdc;
+
+$loader-size: 30px; 
+
+$loader-gutter: $loader-size/10;
+$loader-transform: $loader-size + $loader-gutter * 2;
+
+.loader {
+  border-radius: $loader-size/2;
+  border: $loader-size * .75 solid transparent;
+  height: $loader-size;
+  position: relative;
+  width: $loader-size * 2;
+  background: white;
+  margin: 0 auto 30px;
+}
+
+%loader-dot {
+  animation: pink 1.05s ease-in-out infinite;
+  border-radius: 100%;
+  height: $loader-size;
+  position: absolute;
+  top: 0;
+  width: $loader-size;
+}
+
+.dot-pink {
+  @extend %loader-dot;
+  background: $flickr-pink;
+  left: -$loader-gutter;
+}
+
+.dot-blue {
+  @extend %loader-dot;
+  animation-name: blue;
+  background: $flickr-blue;
+  right: -$loader-gutter;
+}
+
+@keyframes pink {
+    0% {
+      z-index: 1;
+    }
+    50% {
+      transform: translateX($loader-transform);
+    }
+}
+
+@keyframes blue {
+    50% {
+      transform: translateX(-$loader-transform);
+    }
+}
+
+.photo-grid {
+  margin-top: 15px;
+}
+
+a.thumbnail_photo {
+  -webkit-transform: scale(1);
+  transform: scale(1);
+  -webkit-transition: .1s ease-in-out;
+  transition: .1s ease-in-out;
+}
+
+a.thumbnail_photo:hover,
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `thumbnail_photo` should be written in lowercase with hyphens